atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -126,6 +131,28 @@ def generate_auto_labels(self, auto_labels):
         docker_info = self.tasker.get_info()
         generated['com.redhat.build-host'] = docker_info['Name']
 
+        info_url = self.info_url_format
+        labels = Labels(self.labels)
+        find_subs = re.finditer('{([^}]+)}', info_url)
```

Comment:
This looks a lot like a python format string, maybe we could use `self.info_url_format.format(**kwargs)`, where `kwargs` would contain a list of labels?